core/src/staked_validators_cache.rs

```diff
@@ -368,18 +403,77 @@ mod tests {
         (node_keypair_map, vahm)
     }
 
-    #[test_case(325_000_000_u64, 1_usize, 0_usize, 10_usize, 123_u64, Protocol::UDP)]
-    #[test_case(325_000_000_u64, 3_usize, 0_usize, 10_usize, 123_u64, Protocol::QUIC)]
-    #[test_case(325_000_000_u64, 10_usize, 2_usize, 10_usize, 123_u64, Protocol::UDP)]
-    #[test_case(325_000_000_u64, 10_usize, 10_usize, 10_usize, 123_u64, Protocol::QUIC)]
-    #[test_case(325_000_000_u64, 50_usize, 7_usize, 60_usize, 123_u64, Protocol::UDP)]
+    #[test_case(
+        325_000_000_u64,
+        1_usize,
+        0_usize,
+        10_usize,
+        123_u64,
+        Protocol::UDP,
+        false
+    )]
+    #[test_case(
+        325_000_000_u64,
+        3_usize,
+        0_usize,
+        10_usize,
+        123_u64,
+        Protocol::QUIC,
+        false
+    )]
+    #[test_case(
+        325_000_000_u64,
+        10_usize,
+        2_usize,
+        10_usize,
+        123_u64,
+        Protocol::UDP,
+        false
+    )]
+    #[test_case(
+        325_000_000_u64,
+        10_usize,
+        10_usize,
+        10_usize,
+        123_u64,
+        Protocol::QUIC,
+        false
+    )]
+    #[test_case(
+        325_000_000_u64,
+        50_usize,
+        7_usize,
+        60_usize,
+        123_u64,
+        Protocol::UDP,
+        false
+    )]
+    #[test_case(
+        325_000_000_u64,
+        1_usize,
+        0_usize,
+        10_usize,
+        123_u64,
+        Protocol::UDP,
+        true
+    )]
+    #[test_case(
+        325_000_000_u64,
+        50_usize,
+        7_usize,
+        60_usize,
+        123_u64,
+        Protocol::UDP,
```

Comment:
Is the plan to move to QUIC for AG votes at some point? We will likely move to QUIC for tower BFT votes before AG goes live

Reply:
Yes, move to QUIC or SNP. Alex thought we should move to SNP because it allows upgrade better, but guess we will wait a bit to see how fast SNP moves.